src/main/java/team/silvertown/masil/masil/validator/MasilValidator.java

```diff
@@ -28,11 +28,19 @@ public static void validateUrl(String url) {
         }
     }
 
+    public static void validateMasilOwner(Masil masil, User user) {
+        User owner = masil.getUser();
+
+        notNull(owner, MasilErrorCode.NULL_USER);
+        throwIf(!Objects.equals(owner, user),
```

Comment:
`!Objects.equals(owner, user)` 여기에 널 체크가 포함되는데 이미 34번 행에서 처리하니까 아래와 같이 가시죠

```suggestion
        notNull(owner, MasilErrorCode.NULL_USER);
        throwIf(!owner.equals(user),
```

Reply:
둘 다 null이면 true가 돼서 하나만 넣어줬어요